features/step_definitions/user_can_create_account_steps.rb

```diff
@@ -0,0 +1,16 @@
+
+Given("I visit the {string} page") do |string|
+  visit root_path
```

Comment:
You do not use the string here, why?

Perhaps rethink this step

Reply:
Done, removed and changed after feature test changed